docs/views.rst

```diff
@@ -67,4 +75,5 @@ the render function:
     from bottery.message import render
 
     def hello(message):
-        return render(message, 'hello.md', {'question': 'interested, yet?'})
+        return render(message, 'hello.md',
+                      {'question': 'interested, yet?'})
```

Comment:
No need to break the line here.

Reply:
I did this so we wouldn't have to scroll this function on docs